nipype/interfaces/io.py

```diff
@@ -123,6 +123,35 @@ def add_traits(base, names, trait_type=None):
     return base
 
 
+def get_head_bucket(s3_resource, bucket_name):
```

Comment:
Minor quibble: I would put a `_` prefix on the function still, just to mark it as a private function that external code shouldn't rely on.